drivers/sensor/adltc2990/Kconfig

```diff
@@ -0,0 +1,49 @@
+# ADLTC2990 Quad I2C Voltage, Current and Temperature sensor configuration options
+
+# Copyright(c) 2023 Carl Zeiss Meditec AG
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ADLTC2990
+	bool "ADLTC2990 Quad I2C Voltage, Current and Temperature Monitor"
+	default n
+	depends on DT_HAS_ADI_ADLTC2990_ENABLED
+	select I2C
+	help
+	  Enable the driver for Analog Devices LTC2990
+	  Quad I2C Voltage, Current and Temperature Monitor.
+
+if ADLTC2990
+
+choice
+	prompt "Temperature Format"
+
+config ADLTC2990_TEMPERATURE_IN_KELVIN
+	bool "Kelvin"
+
+config ADLTC2990_TEMPERATURE_IN_CELCIUS
+	bool "Celcius"
+
+endchoice
+
+config ADLTC2990_TEMPERATURE_FORMAT_VALUE
+	int
+	default 0 if ADLTC2990_TEMPERATURE_IN_CELCIUS
+	default 1 if ADLTC2990_TEMPERATURE_IN_KELVIN
+
+choice
+	prompt "Acquisition Type"
+
+config ADLTC2990_ACQUISITION_SINGLE
+	bool "Single"
+
+config ADLTC2990_ACQUISITION_REPEATED
+	bool "Repeated"
+
+endchoice
+
+config ADLTC2990_ACQUISITION_TYPE_VALUE
+	int
+	default 0 if ADLTC2990_ACQUISITION_REPEATED
+	default 1 if ADLTC2990_ACQUISITION_SINGLE
```

Comment:
These correspond directly to hardware register fields; move to devicetree properties.

Reply:
done